components/DynamicContent/src/index.scss

```diff
@@ -117,6 +158,34 @@
   word-break: break-word; // Fixes hypens issue for Chrome & safari on windows device.
 }
 
+.denhaag-dynamic-content__card-icon {
+  opacity: var(--denhaag-dynamic-content-icon-opacity);
+  height: var(--denhaag-dynamic-content-icon-size, var(--denhaag-space-block-xl));
+  width: var(--denhaag-dynamic-content-icon-size, var(--denhaag-space-block-xl));
+  color: var(--denhaag-dynamic-content-icon-color);
+  position: relative;
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```

Comment:
Target specific changes. Now you make the transition heavier for no reason. And sort alphabetically